features/FxA-I-11/README.md

```diff
@@ -0,0 +1,128 @@
+# Match 1 - Control vs. No Show
+> Eye Password Experiment FXA-80
+
+Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeeley/eye-password-emoji/features/FxA-80-eye-password-experiment/README.md)
+
+Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
+
+## Outcomes
+
+The success of this feature is being able to answer the question:
+
+"Is the 'Show' password button useful in its current form?".
+"Does it reduce the number of errors the users make on the change password page?"
+
+## Assumptions
+
+- Users use this functionality on Android more.
+- We believe the "Show" password button is useful.
```

Comment:
Update: I think you're right. Match 1 is more about testing our current assumption that the "Show password button is useful", so this makes sense to include that here.